### DISCUSSION TITLE
Helppp

### DISCUSSION THREAD

**Discussion by Anshuman2259** [fast4x/RiMusic#3676] [Q&A]
How do I remove the background timeline 
![Screenshot_20240915-155351](https://github.com/user-attachments/assets/48b0b882-6459-4ccf-90e7-a5086288fa98)

**Comment by fast4x** [comment]
![Screenshot_2024-09-15-14-16-32-607_it.fast4x.rimusic-edit.jpg](https://github.com/user-attachments/assets/43456e05-9b1a-4b67-9276-0e35516f7911)

**Reply by Anshuman2259** [reply]
Not that brother 
The light layer that goes with the timeline, it is behind sthe thumbnail

**Reply by bq69pd** [reply]
Search background progression bar and select disabled.

**Comment by aneesh1122** [comment]
Have you looked into the settings before asking for help?

**Comment by bq69pd** [comment]
Songs are getting automatically downloaded specially when they are on loop. Is there any button to stop(I can't find 1). Thnx

**Reply by fast4x** [reply]
...are automatically cached by player not downloaded. You can launch system cleaner
